src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/ContactByPhone.cshtml.cs

```diff
@@ -22,7 +22,7 @@ public ContactByPhoneModel(IConnectionRequestDistributedCache connectionRequestD
 
     protected override void OnGetWithModel(ConnectionRequestModel model)
     {
-        if (!HasErrors && model.ReferrerContact != null)
+        if (!Errors.HasErrors && model.ReferrerContact != null)
```

Comment:
Use model.ReferrerContact is not null perhaps?

Reply:
I should put in a ticket to just replace a load of these so you don't have to comment this on every PR 🤣